frontend/packages/client/src/components/CommunityCreate/StepTwo.js

```diff
@@ -144,7 +144,43 @@ export default function StepTwo({
         autoFocusOnLoad={true}
       />
       <CommunityUsersForm
-        title="Authors"
+        title={
+          <>
+            Authors
+            <div className="popover is-popover-bottom">
+              <button
+                className="delete has-text-grey rounded-full cursor-pointer popover-trigger"
+                style={buttonStyle}
+              >
+                ?
+              </button>
+              <div className="popover-content">
+                <div className={popoverClassName}>
+                  <div className="column is-12 p-0">
+                    <p
+                      className="small-text has-text-weight-normal has-text-grey small-text mb-1"
+                      style={{
+                        lineHeight: '20px',
+                      }}
+                    >
+                      Author addresses will be added automatically as members
+                      for the community.
+                    </p>
+                    <p
+                      className="small-text has-text-weight-normal has-text-grey small-text"
+                      style={{
+                        lineHeight: '20px',
+                      }}
+                    >
+                      In addition, community creator address will be set as
+                      author and member by default.
+                    </p>
+                  </div>
+                </div>
+              </div>
```

Comment:
Instead of sending 40 lines of code as a prop to component this should signal that you need to convert this into a component or modify the component to accept a composable design structure. For example:
```
// example 1
<CommunityUserForm title={<UserTitle />}

// example 2
<CommunityUserForm>
   <CommunityUserForm.Title props={} />
</CommunityUserForm>

// example 3
<CommunityUserForm>
    <CommunityUserForm.Title>
     {/* title goes here */}
    </CommunityUserForm.Title>
</CommunityUserForm>